cmd/util/ledger/migrations/cadence_values_migration.go

```diff
@@ -209,10 +209,22 @@ func (m *CadenceBaseMigration) MigrateAccount(
 			m.nWorkers,
 		)
 
+		owner := string(address[:])
```

Comment:
Maybe use `AddressToRegisterOwner` to be consistent for global registers.

```suggestion
		owner := AddressToRegisterOwner(address)
```

Reply:
I changed it in [9a2541a](https://github.com/onflow/flow-go/pull/5970/commits/9a2541a2676510f46db8c2e7000fd2a30a4168d3), but is it even required? We call this in `MigrateAccount`, which is only called for accounts, and not for global registers?